packages/core/http/core-http-resources-server-internal/src/http_resources_service.ts

```diff
@@ -155,6 +155,16 @@ export class HttpResourcesService implements CoreService<InternalHttpResourcesSe
           },
         });
       },
+      renderCss(options: HttpResourcesResponseOptions) {
+        return response.ok({
+          body: options.body,
+          headers: {
+            ...options.headers,
+            'content-type': 'text/css',
+            'content-security-policy': cspHeader,
+          },
+        });
+      },
```

Comment:
I strongly suspect there is an easier way to load a CSS, I did this to follow the pattern in the `routes` but I am open to suggestions

Reply:
I'd like to get confirmation from the rest of the @elastic/kibana-core team, but I'd say it's great to add this since we already have `renderJs` and `renderHtml`. My only ask would be to add tests :)

Reply:
Yea, as long as it's tested it sounds fine.

Outside of the scope of this PR but we may want to implement a `render` function accepting a contentType parameter at some point instead of adding a new API for each type of content we want to render 🙈